src/Sylius/Bundle/UserBundle/Command/ChangePasswordCommand.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Sylius\Component\User\Model\UserInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @author Loïc Frémont <loic@mobizel.com>
+ */
+class ChangePasswordCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('sylius:user:change-password')
+            ->setDescription('Change the password of a user.')
+            ->setDefinition(array(
+                new InputArgument('email', InputArgument::REQUIRED, 'The email'),
+                new InputArgument('password', InputArgument::REQUIRED, 'The password'),
+            ))
+            ->setHelp(<<<EOT
+The <info>sylius:user:change-password</info> command changes the password of a user:
+  <info>php app/console sylius:user:change-password matthieu</info>
```

Comment:
Replacing `matthieu` with `matthieu@email.com` will be more explicit to the user